db/migrate/20230517212721_add_enqueue_to_parties.rb

```diff
@@ -0,0 +1,5 @@
+class AddEnqueueToParties < ActiveRecord::Migration[6.0]
+  def change
+    add_column :parties, :enqueued, :boolean
```

Comment:
For the boolean fields, we generally want to set a default to prevent three states of a boolean column (`null`, `true` and `false`). In this case we would want it to be `false`. You can read more about [it here](https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem)

```suggestion
    add_column :parties, :enqueued, :boolean, default: false
```